sandpack-react/src/components/Tests/Header.tsx

```diff
@@ -87,24 +93,28 @@ export const Header: React.FC<Props> = ({
             Suite only
           </button>
         )}
-        <button
-          className={buttonsClassName}
-          data-active={verbose}
-          disabled={status === "initialising"}
-          onClick={setVerbose}
-          type="button"
-        >
-          Verbose
-        </button>
-        <button
-          className={buttonsClassName}
-          data-active={watchMode}
-          disabled={status === "initialising"}
-          onClick={setWatchMode}
-          type="button"
-        >
-          Watch
-        </button>
+        {!hideVerboseButton && ! (
+          <button
+            className={buttonsClassName}
+            data-active={verbose}
+            disabled={status === "initialising" ||sandpack.hideTestsAndSupressLogs}
```

Comment:
If `hideTestsAndSupressLogs` is only used on the Test component, I wouldn't make it part of the Sandpack context, but a prop for `SandpackTests`

Reply:
Sure! Changing it :)